scanner.js

```diff
@@ -169,7 +169,10 @@ FirefoxOnAndroidRuntime.prototype = Object.create(Runtime.prototype);
 
 Object.defineProperty(FirefoxOnAndroidRuntime.prototype, "name", {
   get() {
-    let packageName = this._socketPath.split("/")[3];
+    // If using abstract socket address, it is "@org.mozilla.firefox/..."
+    let packageName = this._socketPath.startsWith("@") ?
```

Comment:
This should be 'const' otherwise lint tools will complain  when this change gets merged into devtools.